docs/page-objects/retries.rst

```diff
@@ -1,88 +0,0 @@
-.. _retries:
```

Comment:
Have you moved this all to the input-validation section because we recommend raising Retry from validate_input method, preferring it to to_item? That's a nice touch.